packages/react/src/components/TeachingBubble/TeachingBubbleContent.base.tsx

```diff
@@ -36,6 +36,12 @@ export const TeachingBubbleContentBase: React.FunctionComponent<ITeachingBubbleP
   const documentRef = useDocument();
   const mergedRootRef = useMergedRefs(rootElementRef, forwardedRef);
 
+  const ariaDescribedById = useId('teaching-bubble-content-');
+  const ariaLabelledById = useId('teaching-bubble-title-');
+
+  const ariaDescribedBy = props.ariaDescribedBy ?? ariaDescribedById;
+  const ariaLabelledBy = props.ariaLabelledBy ?? ariaLabelledById;
```

Comment:
nit: We could avoid generating an id if there's already an `ariaDescribedBy` or `ariaLabelledBy`
```suggestion
  const ariaDescribedBy = props.ariaDescribedBy ?? useId('teaching-bubble-content-');
  const ariaLabelledBy = props.ariaLabelledBy ??  useId('teaching-bubble-title-');
```

Reply:
We cannot. `useId` is a hook function and using `??` would result in calling it conditionally which makes daddy Facebook unhappy.